src/components/thumbnail-card/ThumbnailCard.scss

```diff
@@ -15,8 +15,7 @@
 
 // We should apply border to the card only when it is hovered
 .thumbnail-card.is-highlight-applied {
-    &:focus,
```

Comment:
Should not apply blue border when the card is focused because there is a little janky UI when there is the orange focus border on to of the blue border.

Reply:
I'd recommend creating a Storybook story and image snapshot for this component, to ensure that there are no visual regressions.